addon/TelemetrySender.js

```diff
@@ -63,6 +63,8 @@ TelemetrySender.prototype = {
   },
 
   _sendPing(data) {
+    console.log(`PAYLOAD: ${data}`);
```

Comment:
Just checking, we explicitly want to keep this? You mention it in the documentation, and this is a separate branch anyway.